### PR TITLE
fix(mobile): error message appear authorized user tap on "Tasks" bb-350

### PR DESCRIPTION
**Pull request by KotenkoNata**
Fixed an error message that appears when an authorized user taps on the "Tasks" button.

[task_fix.webm](https://github.com/user-attachments/assets/3bf941bd-d0d7-4f0c-b076-7addabb02ee5)

### CODE REVIEW

File: apps/mobile/src/packages/users/users.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export { UserValidationMessage } from "./libs/enums/enums";
 export {
 	type UserDto,
 	type UserGetAllResponseDto,
```
Comment (fshabanov) [inline apps/mobile/src/packages/users/users.ts:17]: Can we completely remove UserGetAllResponseDto or is it still used somewhere?

File: apps/mobile/src/slices/users/actions.ts [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
-const loadAll = createAsyncThunk<
-	UserGetAllResponseDto,
-	undefined,
+const getById = createAsyncThunk<
+	UserDto,
+	UserGetParametersDto,
 	AsyncThunkConfig
->(`${sliceName}/load-all`, (_, { extra }) => {
+>(`${sliceName}/get`, (payload, { extra }) => {
```
Comment (fshabanov) [inline apps/mobile/src/slices/users/actions.ts:15]:
```suggestion
>(`${sliceName}/get-by-id`, (payload, { extra }) => {
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by revenets — APPROVED** [review]

File: apps/mobile/src/packages/users/users.ts (hunk 1 above)
Comment (fshabanov) [reply apps/mobile/src/packages/users/users.ts:17]: If only in Backend, then you can remove from mobile?

**Review by fshabanov — APPROVED** [review]

File: apps/mobile/src/packages/users/users.ts (hunk 1 above)
Comment (KotenkoNata) [reply apps/mobile/src/packages/users/users.ts]: Done

**Pull request merged**